### ISSUE TITLE
Memory Leak Resolved in latest commit?

### ISSUE THREAD

**Issue by thicccatto** [expired6978/SKSE64Plugins#52]
Hi, I noticed large changes made regarding the architecture of RaceMenu. Does this resolve the memory leak?

**Comment by thicccatto** [comment]
Ive been trying to compile for 2 hours, but there are no instructions available?

**Comment by darkconsole** [comment]
yeah the lack of instructions, documentation, and or working templates, is one of my favourite facets of the skse dll world.

**Comment by thicccatto** [comment]
> yeah the lack of instructions, documentation, and or working templates, is one of my favourite facets of the skse dll world.

I've never seen code or a Dev like this before. No comments, no docs, doesn't address issues.

Do you reckon you could have a go at compiling it or give me some steps? I managed to find all the dependencies but visual studio compilation is alien to me.
